packages/plexus/src/LayoutManager/LayoutManager.tsx

```diff
@@ -17,17 +17,17 @@ import { TCancelled, TEdge, TLayoutDone, TPendingLayoutResult, TPositionsDone, T
 
 import Coordinator from './Coordinator';
 
-type TPendingResult<T> = {
+type TPendingResult<T, U> = {
   id: number;
   isPositionsResolved: boolean;
   resolvePositions?: (result: TCancelled | TPositionsDone<T>) => void;
-  resolveLayout?: (result: TCancelled | TLayoutDone<T>) => void;
+  resolveLayout?: (result: TCancelled | TLayoutDone<T, U>) => void;
 };
 
-export default class LayoutManager<T = {}> {
+export default class LayoutManager {
   layoutId: number;
-  coordinator: Coordinator<T>;
-  pendingResult: TPendingResult<T> | null;
+  coordinator: Coordinator;
+  pendingResult: TPendingResult<any, any> | null;
```

Comment:
why were the generics here and in `_handleUpdate` replaced with `any`? does it simply not matter for LayoutManager?

Reply:
Correct, it's not relevant to the `LayoutManager` which only needs to work with the data on `TEdge` and `TSizeVertex`.